### DISCUSSION TITLE
AVG flagging the download as malware and unable to complete setup

### DISCUSSION THREAD

**Discussion by TxTitan27** [KRTirtho/spotube#1554] [Q&A]
I mark it as an exception and remove it from quarantine, but it doesn't like it. DO I get rid of AVG and find another virus software?

**Comment by KRTirtho** [comment]
I'm not gonna recommend any Anti-Virus or anything like that but from my experience I can gurantee you any free Anti-virus software except Microsoft's Official Windows Defender is crap and basically adwares. I'm pretty sure some of them are actually worse than malwares

**Comment by jedooz** [comment]
Hello
Unfortunately Windows Defender detect Spotube as a virus :
PUA:Win32/Packunwan
